### ISSUE TITLE
Unexpected argument type at index 0

### ISSUE THREAD

**Issue by Globegitter** [derhuerst/vibrant#1]
Hey there, just started using that theme and I like it. But I am getting an error whenever I go to one of my git repositories. No matter what I do or type I get:
```
test: Unexpected argument type at index 0
test: Missing argument at index 2
test: Missing argument at index 2
```

If I switch theme all is fine.

**Comment by derhuerst** [comment]
You're right. I have get the same error when using it. Can you narrow down when this happens for you? I noticed that error occurs whenever I'm in detached `HEAD` mode in a Git repo.

Will fix it soon, as it bugs me as well.

**Comment by Globegitter** [comment]
Hmm, that is odd, I temporarily switched to another theme to get rid of this error and now trying to switch back (`fisher install vibrant`) nothing seems to happen. I can switch to other themes. Not sure if this is an issue with this theme or fisherman.

**Comment by derhuerst** [comment]
Should be fixed as of 0a770dd.

**Comment by Globegitter** [comment]
So, finally managed to switch back to this theme. Will test a bit more and close if it indeed has gone away. Thanks!
